openfl/interface/director.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+import logging
+
+from click import group, pass_context
+from click import option
+from openfl.component.director.director import serve
+
+logger = logging.getLogger(__name__)
+
+
+@group()
+@pass_context
+def director(context):
+    """Manage Federated Learning Director."""
+    context.obj['group'] = 'director'
+
+
+@director.command(name='start')
+@pass_context
+@option('--sample-shape', '-ss', multiple=True,
+        help='Sample shape')
+@option('--target-shape', '-ts', multiple=True,
+        help='Target shape')
+def start_(context, sample_shape, target_shape):
```

Comment:
We probably want to choose which port the server will be running from.
Or, if a director's server will choose a port by itself, it should log the port.

Reply:
Done